src/App.css

```diff
@@ -23,11 +23,23 @@ ul {
 .screen {
   width: 100%;
   border: 1px solid black;
+  height: auto;
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  justify-content: flex-end;
 }
```

Comment:
- [OPTIONAL] Make The Calculator.js component match the following given design:
![Screenshot from 2021-11-03 14-05-23](https://user-images.githubusercontent.com/40503704/140065247-fe9c0796-ab2f-4769-8faa-81f548dcfeaf.png)

Reply:
i'll check that